src/iap/modules/auth/components/signup-form/signup-form.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+import { ApiResponse } from '@shared/models/auth.models';
+import { IapError } from '@shared/models/types/iap-error.type';
+import { IapSyncValidators } from '@shared/validators/iap-sync-validators.validator';
+import { IsLoginUniqueValidator } from '@shared/validators/is-login-unique.validator';
+import { AuthFormsApiService } from '../../services/auth-forms-api.service';
+
+@Component({
+	selector: 'iap-signup-form',
+	templateUrl: './signup-form.component.html',
+	styleUrls: ['./signup-form.component.scss'],
+})
+export class SignupFormComponent implements OnInit {
+	@Output() submitEvent: EventEmitter<ApiResponse> = new EventEmitter();
```

Comment:
I think we have already discussed this topic, but when your variable can define its type automatically you haven't to add pure typing.
```suggestion
	@Output() submitEvent = new EventEmitterr<ApiResponse>();
```